payments/credit-card/index.md

```diff
@@ -0,0 +1,514 @@
+
+#### Type of capture (Intent)
+
+* **AutoCapture (one-phase):** If you want the credit card to be charged right away, you will have to specify that the `intent` of the purchase is `AutoCapture`. The credit card will be charged and you don't need to do any more financial operations to this purchase.
+
+#### General
+
+* **No 3D Secure and card acceptance**: There are optional paramers that can be used in relation to 3d-secure and card acceptance. By default, most credit card agreements with an acquirer will require that you use 3D-Secure for card holder authentication. However, if your agreement allows you to make a card payment without this authentication, or that specific cards can be declined, you may adjust these optional parameters when posting in the payment. This is specified in the technical reference section for creating credit card payments  - you will find the link in the sequence diagram below.
+* **Defining CallbackURL**: When implementing a scenario, it is optional to set a [CallbackURL](/xwiki/wiki/developer/view/Main/ecommerce/technical-reference/core-payment-resources/#HURLs) in the `POST` request. If callbackURL is set PayEx will send a postback request to this URL when the consumer has fulfilled the payment. [See the Callback API description here.](/xwiki/wiki/developer/view/Main/ecommerce/technical-reference/#HCallback)
+
+## Purchase flow
+
+The sequence diagram below shows a high level description of a complete purchase, and the two requests you have to send to PayEx. The links will take you directly to the corresponding API description.
+
+When dealing with credit card payments, 3D-Secure authentication of the cardholder is an essential topic. There are three alternative outcome of a credit card payment:
+
+* 3D-Secure enabled - by default, 3D-secure should be enabled, and PayEx will check if the card is enrolled with 3D-secure. This depends on the issuer of the card. If the card is not enrolled with 3D-Secure, no authentication of the cardholder is done.
+* Card supports 3D-Secure - if the card is enrolled with 3D-Secure, PayEx will redirect the cardholder to the autentication mechanism that is decided by the issuing bank. Normally this will be done using BankID or Mobile BankID. 
+
+<embed src="https://developer.payex.com/xwiki/wiki/developer/get/Main/ecommerce/payex-payment-instruments/card-payments/card-payment-pages-in-mobile-apps/WebHome?xpage=plain&amp;uml=1" style="max-width:100%">
+
+### Detailed Sequence Diagram enabing 3D-secure authentication
+
+<embed src="https://developer.payex.com/xwiki/wiki/developer/get/Main/ecommerce/payex-payment-instruments/card-payments/card-payment-pages-in-mobile-apps/WebHome?xpage=plain&amp;uml=2" style="max-width:100%">
+
+## Options after posting a payment
```

Comment:
Seems like some duplication of content here? Why is this section showing up again?